tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/codegeneration/cpp/CppViewHelper.java

```diff
@@ -140,7 +140,7 @@ public static String computeDefaultValue(Shape shape) {
     }
 
     public static String computeVariableName(String memberName) {
-        return memberName.substring(0, 1).toLowerCase() + memberName.substring(1);
+        return lowercasesFirstChar(memberName).replace("-", "_");
```

Comment:
instead of underscores/string manipulation should we just copy what we do for [upper camel case](https://github.com/aws/aws-sdk-cpp/blob/main/tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/codegeneration/cpp/CppViewHelper.java#L146-L148), but make it for lower camel case? since its _only variable_ names shouldn't be publicly impacting, but it could make a nasty delta.

Reply:
well, the issue with `CaseFormat` is that we need to know the source case convention, but we don't know it.

Reply:
wouldn't it always be lower camelcase?

Reply:
we don't control it, it comes from the model file.